autogen/cluster.tf

```diff
@@ -233,6 +233,8 @@ locals {
   ]
 }
 
+# This keepers list is based on the terraform google provider schemaNodeConfig
+# resources where "ForceNew" is "true"
```

Comment:
It'd be good to have a direct link to the code as well.

Reply:
added a link to the source.